exercises/list-ops/.meta/additional_tests.json

```diff
@@ -0,0 +1,32 @@
+{
+  "cases": [
+    {
+      "description": "fold additional cases",
+      "cases": [
```

Comment:
Any reason we can't just single-nest these here?

Reply:
There is no reason not to single nest these tests definitions. I just kept to the format of canonical-data. 
Should I unfold them ?

Reply:
I unfolded them...